Feature/#4 entity

kos24 opened the pull request:
#4: added entities and repositories

erenkov committed:
#4: add entity: task, release, taskkRelease. pom.xml changed

erenkov committed:
#4: add entity: task, release, taskkRelease. pom.xml changed +

erenkov committed:
add Composite Primary Keys to TaskReleaseEntity

sifdragon committed:
Added enum entities (#7)

* Added enum entities

* Added TaskType enum

Co-authored-by: IlnazG <wistfuldr@mail.com>

sifdragon committed:
Merge pull request #8 from erenkov/subtask/#4-gilmutdinov-2

Updated enum's + added converters for enum's

Commit:
Updated enum's + added converters for enum's

kos24 committed:
#4: added entities

kos24 committed:
Merge branch 'feature/#4-entity' into subtask/#4-rebrin

kos24 committed:
Merge pull request #9 from erenkov/subtask/#4-rebrin

Subtask/#4 rebrin

kos24 committed:
#3: added liquibase ddl script

kos24 committed:
added OneToMany relationships

kos24 committed:
Merge pull request #14 from erenkov/bugfix/#4-entity

added OneToMany relationships and fixed entities

kos24 committed:
#3: changed changelog-1.0

kos24 committed:
#3: minor fixes in ddl script

kos24 committed:
#4: added Converter annotation to enum classes

kos24 committed:
#4: added constructor to ProjectEntity

hotabsc commented on src/main/java/com/developing/simbir_product/entity/ProjectEntity.java:
```diff
@@ -0,0 +1,142 @@
+    @GeneratedValue(generator = "UUIDGenerator", strategy = GenerationType.AUTO)
+    @Column(name = "id", updatable = false, nullable = false)
+    @Type(type="org.hibernate.type.PostgresUUIDType")
+    private UUID id;
```
а разве вот так по-простому не будет автоматически генерировать?
`    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private UUID id;
    `

kos24 replied:
делал на основе статей из интернета, из самых свежих попалась такая реализация, попробуем по-простому

hotabsc commented:
мержить после исправлений #17

kos24 committed:
#4: added String.format to exception message

kos24 committed:
#4: changed table USER and some annotations

kos24 committed:
#3: changed table name for USR and changed field to pass

Nikitinmn42 committed:
All LocalDateTime fields in entity replaced by OffsetDateTime

Nikitinmn42 committed:
#3: Added user_number column to usr table

kos24 committed:
# This is a combination of 3 commits.
# This is the 1st commit message:

#3 added FK for user_task_history

# This is the commit message #2:

#3: changed table name for task_release_history

# This is the commit message #3:

#3:changed liquibase: rename table task_release_history, added FK

kos24 committed:
#3 added FK for user_task_history

kos24 committed:
#3: changed table name for task_release_history

kos24 committed:
#3:changed liquibase: rename table task_release_history, added FK

kos24 committed:
#4:renamed class for TaskReleaseHistoryEntity

kos24 committed:
#3: added surrogate keys for User_Task_History and Task_Release_History

kos24 committed:
Merge branch 'bugfix/#3-add-usernumber' of https://github.com/erenkov/project-manager into bugfix/#3-add-usernumber

kos24 committed:
#4: changed TaskReleaseHistoryEntity and UserTaskHistoryEntity

kos24 committed:
Merge branch 'feature/#4-entity' into bugfix/#4-entity

kos24 committed:
#4: changed TaskReleaseHistoryEntity and UserTaskHistoryEntity

kos24 committed:
#4: minor fixes

kos24 committed:
#4: Minor fixes

kos24 committed:
#4: Deleted TaskReleaseId and UserTaskId

erenkov committed:
#4: fixed entity: table-lowerCase, userNumber => teamId, del comments

erenkov committed:
#4: delited teamId from constructor userEntity

Nikitinmn42 committed:
Merge pull request #20 from erenkov/bugfix/#4-entity

Bugfix/#4 entity

Nikitinmn42 committed:
Merge pull request #19 from erenkov/bugfix/#3-add-usernumber

#3: Added user_number column to usr table

kos24 committed:
Merge branch 'feature/#3-liquibase' into feature/#4-entity
